plugins/woocommerce/client/admin/client/settings-payments/components/incentive-modal/incentive-modal.tsx

```diff
@@ -146,10 +158,10 @@ export const IncentiveModal = ( {
 										disabled={ isBusy }
 										onClick={ () => {
 											setIsBusy( true );
-											// TODO: Temporary for testing, update to use plugin ID and slug.
-											onAccept(
-												'woopayments',
-												'woocommerce-payments'
+											onAccept( incentive.id );
```

Comment:
Nitpick: Probably a good idea to extract this method as its gotten taller.

Reply:
Good idea. Updated in a0ecc32cec!